.github/workflows/publish-npm-packages.yaml

```diff
@@ -29,6 +29,8 @@ jobs:
     environment: publish-npm
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
```

Comment:
Not sure what we change but the publishing to npm step complains we are not in a git repo and it's broken in main 🤷‍♂️ :

<img width="1333" alt="Screenshot 2025-04-22 at 11 28 56" src="https://github.com/user-attachments/assets/7e19733e-76cd-484e-9451-0a1eb429395a" />

Reply:
Would make sense to set to `onLanguage:cypher` maybe? 

https://code.visualstudio.com/api/references/activation-events#onLanguage